MobileCenterCrashes/MobileCenterCrashes/MSCrashes.m

```diff
@@ -187,7 +187,8 @@ - (void)applyEnabledState:(BOOL)isEnabled {
     if (self.crashFiles.count > 0) {
       [self startDelayedCrashProcessing];
     }
-    MSLogInfo([MSCrashes getLoggerTag], @"Crashes service has been enabled.");
+    if (![MSMobileCenter isDebuggerAttached])
+      MSLogInfo([MSCrashes getLoggerTag], @"Crashes service has been enabled.");
```

Comment:
We have coding styles for that. If any statements have one-line body, we don't use curly braces unless it is not easy to read. I know there are some statements with curly braces, we are currently mix-using them. We will have an internal discussion and make it consistent.